spec/inquisition/brakeman/runner_spec.rb

```diff
@@ -1,4 +1,38 @@
+require 'ruby_parser/bm_sexp_processor'
+require 'brakeman/processors/lib/safe_call_helper'
+require 'brakeman/warning'
+require 'brakeman/checks/base_check'
+
 RSpec.describe Inquisition::Brakeman::Runner do
+  let(:runner) { described_class.new }
+  let(:high_confidence_warning) { Brakeman::Warning.new(confidence: 0) }
```

Comment:
Propose to use [`instance_double`](https://relishapp.com/rspec/rspec-mocks/v/3-1/docs/verifying-doubles/using-an-instance-double)